src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+  
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+  
+.stars__star:nth-child(-n+4) {
+  background-image: url(./star-active.svg);
+}
```

Comment:
you need to use all active stars conditions in this task, your solutions willn't work with quantity of stars not equal 4